terraform/integrations.tf

```diff
@@ -0,0 +1,189 @@
+# Integrations between Kafka products
```

Comment:
I really like this layout! It is the most fuzzy part of the spec imo, and you handled it quite well considering all the conditionals we allow on deployment. very neat